.ci/jobs/build-docker-images.yml

```diff
@@ -17,6 +22,6 @@
             credentials-id: f6c7695a-671e-4f4f-a331-acdce44ff9ba
             reference-repo: /var/lib/jenkins/.git-references/e2e-testing.git
             branches:
-              - master
+              - $branch_specifier
     triggers:
-      - timed: 'H H(0-5) * * 1-5'
+      - timed: '@daily'
```

Comment:
If it helps you can remove this and use the one in the pipeline, though the very first build might be triggered manually to apply the changes from the pipeline. Again another 🐔 🥚